src/components/CardLists/planner.js

```diff
@@ -0,0 +1,32 @@
+import useStore from '../../hooks/useStore';
+import StyledCard from '../Card/styled';
+import StyledHeadline from '../CardHeadline/styled';
+
+import StyledCardlist from './styled';
+
+function CardlistPlanner() {
+	const cards = useStore(state => state.cards);
+	const deleteCard = useStore(state => state.deleteCard);
+
+	return (
+		<>
+			<p>Ausgewählte Übungen</p>
+			<StyledCardlist>
+				{cards.map(card => {
+					return (
+						<StyledCard
+							key={card.id}
+							onClick={() => {
+								deleteCard(card.id);
+							}}
```

Comment:
So I can delete the card by clicking it where ever I want?

Reply:
yes, the delete button is my next user story. I know, I should have written the delete function and the delete button in the same user story. :(